README.md

```diff
@@ -20,7 +20,7 @@ For the technical details of the proposal, code examples, API shape, etc. see [p
 ## Terminology Used
 
 ###### Agent
-An autonomous assistant that can understand a user's goals and take actions on the user's behalf to achieve them. Today,
+An autonomous assistant that can understand user's goals and take actions on the user's behalf to achieve them. Today,
```

Comment:
Here the indefinite article "a" refers to the possessive noun ”user’s”, and as such the original text seems correct to me. But I’ll let some native speaker to chime in.